src/background.js

```diff
@@ -12,44 +12,65 @@ LICENSE: GPL-2.0
 
 */
 
+const NOTIFICATION_AUTOHIDE_DELAY = 700;
+
 var readLaterObject = readLater(chrome.storage.sync);
 
-var readLaterApp = (function(){
-	var add_success = function(){
-		console.log(`URL Item successfully added.`);
-	};
+var readLaterApp = (function() {
 
-	var add_exists = function(urlItem){
-		var urlJSON = JSON.stringify(urlItem);
-		console.log(`Add failed. URLItem ${urlJSON} already exists.`);
-	};
+	function displayNotification(id, title, message) {
+		chrome.notifications.create(id, {
+			type: "basic",
+			title: title,
+			message: message,
+			iconUrl: "icon.png"
+		}, function(id) {
+			setTimeout(function() {
+				chrome.notifications.clear(id);
+			}, NOTIFICATION_AUTOHIDE_DELAY);
+		});
+	}
 
-	var remove_success = function(){
+	var remove_success = function(removedItem) {
+		displayNotification('RemovedFromBackground', "Page Removed", removedItem.title);		
 		console.log(`URL successfully removed.`);
 	};
 
 	var remove_failed = function(url){
 		console.log(`Remove failed. URL ${url} does not exist.`);
 	};
+	
+	let removeUrlHandler = readLaterObject.removeURLHandler(remove_success, remove_failed)
+
+	var add_success = function(addedItem) {
+		displayNotification('AddedFromBackground', "Page Added", addedItem.data.title);		
+		console.log(`URL Item successfully added.`);
+	};
+
+	var add_exists = function(urlItem) {
```

Comment:
Actually, what this PR does is implement the feature you approved in https://github.com/napsternxg/ReadLater/issues/28.
Changing this logic will defeat its main purpose.

Reply:
Sorry, I forgot about the details. 
Is it possible to rename the button to ADD and REMOVE based on the presence of the URL in the storage?
This way the UI will make it more explicit to the user that the URL will be removed.
Also, if the invocation is by keyboard shortcut then we should ask for user confirmation before item removal.

I am trying to ensure this as many users have contacted me about missed URLs because the app instructions were not clear on some parts. It is important to make the usage of the app as simple and explanatory as possible.

Reply:
I think there's no point to discuss this any more.
Thanks for your time!